docker/docker-compose.yaml

```diff
@@ -1,6 +1,6 @@
 services:
   fedimintd:
-    image: fedimint/fedimintd:v0.8.0
+    image: fedimint/fedimintd:v0.8.1
```

Comment:
should we add the image digest so that this is always deterministic?

Reply:
We've never published the same version tag twice (I'm not sure what CI would do, but I'd expect some failures). I think forcing an image digest on a generic platform like umbrel makes sense, since you can't control the image publishing pipeline for different packages, but I'm not sure it benefits us.

Let me know if you see a strong benefit that I'm overlooking.